shader/src/lib.rs

```diff
@@ -43,7 +43,13 @@ fn shader(v: Vertex, uniforms: &Uniforms, shader_name: &String) -> LinSrgb {
         // "MitchWash" => wash_shaders::mitch_wash::shader(v, uniforms),
         // "ColourPalettes" => wash_shaders::colour_palettes::shader(v, uniforms),
         _ => lin_srgb(0.0, 0.0, 0.0)
-    }
+    };
 
+    if let Light::Wash { .. } = v.light {
+        // TODO: Make this a slider parameter.
+        let lerp_amt = 1.0;
```

Comment:
@JoshuaBatty we should be able to replace this with whatever uniform value you want to use to control the wash light interpolation.